src/openrct2/config/Config.h

```diff
@@ -209,6 +209,14 @@ enum TEMPERATURE_FORMAT
     TEMPERATURE_FORMAT_F
 };
 
+enum SCALE_QUALITY
```

Comment:
You should be able to define this as `enum class` then ditch the `SCALE_QUALITY` prefix.

Reply:
I'm having a little trouble converting it to an enum class, the ConfigEnum/ConfigEnumEntry apparently doesn't understand the format of the enum class and expects an sint32 (casting might work but make it only harder to read). Also, the other settings enums are not enum classes too, just regular enums. Might it be a better idea to convert all those enums to enum classes and update the config to handle them in a separate issue?

Reply:
@IntelOrca this would require either casting the `enum class` to `sint32` (as this is not implicit) or modify `ConfigEnumEntry` to do this automatically (which doesn't sound good to me). I'm happy with this code being merged the way it is now, what is your opinion?

Reply:
OK - nevermind then.